src/main/java/ir/ac/sbu/parser/LLParserGenerator.java

```diff
@@ -66,6 +66,8 @@ public LLParserGenerator(List<GraphModel> graphs) throws TableException {
                 .filter(graphModel -> nullableNodes.contains(graphModel.getStart().getId()))
                 .map(GraphModel::getName)
                 .collect(Collectors.toSet());
+
+        getTableString(30); //for finaly checking for errors
```

Comment:
Thanks for pull request. However this implementation has purpose. we want to only check that graph has no duplicate name, invalid name, invalid connections, etc. The other type of error generated when generate parser table are related to LL1 grammar and it show that your graph which show a grammar, is not LL1. It is different from check graph. Please close this change because it is not necessary.